ui/app/AppLayouts/Wallet/views/collectibles/CollectibleDetailView.qml

```diff
@@ -14,6 +14,7 @@ import AppLayouts.Communities.panels 1.0
 import utils 1.0
 import shared.controls 1.0
 import shared.views 1.0
+import shared.views.chat 1.0
```

Comment:
this feels weird... those dependencies should probably be moved to a more generic location